library/src/main/java/com/proxerme/library/connection/info/entity/Synonym.java

```diff
@@ -0,0 +1,126 @@
+package com.proxerme.library.connection.info.entity;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.support.annotation.NonNull;
+
+import com.afollestad.bridge.annotations.Body;
+
+/**
+ * An entity holding the synonyms of an entry name (Anime, Manga).
+ *
+ * @author Desnoo
+ */
+public class Synonym implements Parcelable {
```

Comment:
This entity is missing the copy- and default (package protected empty) constructor